src/libraries/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -458,6 +466,12 @@ protected virtual Task<Stream> CreateContentReadStreamAsync()
             return WaitAndReturnAsync(LoadIntoBufferAsync(), this, s => (Stream)s._bufferedContent);
         }
 
+        protected virtual Task<Stream> CreateContentReadStreamAsync(CancellationToken cancellationToken)
+        {
+            // Drops the CT for compatibility reasons, see https://github.com/dotnet/corefx/issues/32615#issuecomment-562083237
+            return CreateContentReadStreamAsync();
```

Comment:
I don't think the usability win would be huge here, but we should consider having the default implementation of both methods check to see if either was overloaded and forward to the right one.

Reply:
All BCL http content implementations that can make use of cancellation already do that. Stephen added most, i added that on MultipartContent in this PR